### PR TITLE
Docs contributing conduct file

### LINKED ISSUE #3: [Semaphore  Boilerplate] Issue 3: Create the Code of conduct template and Contributing files
Description:

To foster a welcoming and inclusive environment, the Semaphore Boilerplate project needs a Code of Conduct and a comprehensive Contributing file. These documents will ensure that contributors understand the expectations for participation and the processes for making contributions.

### PR DESCRIPTION
**Pull request by NueloSE**
# Create the Code of conduct template and Contributing files

## 📝 Summary
This documents provides a welcoming and inclusive environment to the new contributors and returning contributors, providing contributors with a comprehensive code of conduct and contribution file. These documents will ensure that contributors understand the expectations for participation and the processes for making contributions.

## 🪧 Related Issues
- Closes #3 

## 🛠️ Type of Change
Mark with an x all the checkboxes that apply (like [x]).

- [x] 📝 Documentation (updates to README, docs, or comments)
- [ ] 🐛 Bug fix (non-breaking change which fixes an issue)
- [ ] 👌 Enhancement (non-breaking change which adds functionality)
- [ ] 💥 Breaking change (fix or feature that would cause existing functionality to change)

## 🔄 Changes Made
Created 2 files:
- CODE_OF_CONDUCT.md: Community guidelines and expected behavior
- CONTRIBUTION.md: Guide for new and returning contributors

## 🔧 Tests Results
No need to run test. The changes implemented  was just documentation

## 📜 Checklist
Please check all the boxes that apply to this PR.

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by evgongora** [comment]
Hello! I think you mistaken both files.

**Comment by evgongora** [comment]
Also could you follow the PR template? We use it for a reason, thank you :)

**Comment by NueloSE** [comment]
@evgongora, thank you for your review. I am unclear about what you meant by "follow the PR template." I may have misunderstood the PR guide.

Here is my interpretation based on the [contribution.md](https://github.com/ZencypherSolutions/semaphore-stellar-docs/blob/main/CONTRIBUTING.md):

I created a branch named docs-contributing-conduct-file, following the contributor guide's example, For documentation: docs-meaningful-name.
Could you kindly guide me on how to improve this further? Thank you.

**Comment by NueloSE** [comment]
> Hello! I think you mistaken both files.

i understand what am to do here

**Comment by evgongora** [comment]
> @evgongora, thank you for your review. I am unclear about what you meant by "follow the PR template." I may have misunderstood the PR guide.
> 
> Here is my interpretation based on the [contribution.md](https://github.com/ZencypherSolutions/semaphore-stellar-docs/blob/main/CONTRIBUTING.md):
> 
> I created a branch named docs-contributing-conduct-file, following the contributor guide's example, For documentation: docs-meaningful-name. Could you kindly guide me on how to improve this further? Thank you.

Hey! @NueloSE I mean this PR template
https://github.com/ZencypherSolutions/semaphore-stellar-boilerplate/blob/main/.github/PR_TEMPLATE/PR_TEMPLATE.md

**Comment by evgongora** [comment]
> > Hello! I think you mistaken both files.
> 
> i understand what am to do here

You made a switch on both docs and placed the code of conduct in the contribution.md as well as the contribution in code of conduct

**Comment by NueloSE** [comment]
@evgongora thank you for your response. Implemented requested changes and ready for review

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+#### Example:
+
+```bash
+git checkout -b docs-installation-guide
+```
+
+### Commit Guidelines
+
+Follow the conventional Commits specification:
```
Comment (evgongora) [inline CONTRIBUTING.md]: Could you add here a link to the conventional commits page?

**Comment by evgongora** [comment]
@NueloSE Hey! Any updates on the review?

**Review by evgongora — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (NueloSE) [reply CONTRIBUTING.md]: okay

**Comment by NueloSE** [comment]
@evgongora  requested changes have been implemented ready for review

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -56,7 +56,7 @@ git checkout -b docs-installation-guide
 
 ### Commit Guidelines
 
-Follow the conventional Commits specification:
+Follow the conventional [Commits specification](https://github.com/ZencypherSolutions/semaphore-stellar-docs/blob/main/CONTRIBUTING.md#commit-your-changes)
```
Comment (evgongora) [inline CONTRIBUTING.md]: I mean this page https://www.conventionalcommits.org/en/v1.0.0/

**Review by evgongora — CHANGES_REQUESTED** [review]

**Comment by NueloSE** [comment]
@evgongora done

**Review by evgongora — APPROVED** [review]
Thank you!

**Pull request merged**